Web/Container/LICENSE

```diff
@@ -0,0 +1,15 @@
+The Apache License v2.0
```

Comment:
Hey, please copy the LICENSE from web-widgets, because this version is a short one.

Reply:
Thanks! I've added that Licence file to all widgets now, including this one